modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/resource/v1_0/test/ObjectEntryResourceTest.java

```diff
@@ -12332,6 +12332,51 @@ private void _testPatchPutCustomObjectEntryWithAttachmentField(
 			testFileEntry, httpMethod, null, objectDefinition,
 			_OBJECT_FIELD_NAME_ATTACHMENT_USER_COMPUTER_SOURCE_2,
 			useExternalReferenceCode);
+
+		// String length limit
```

Comment:
@ccorreagg I think we can get rid of this test since I added it also in vulcan-test. But I want to know what you think.

Reply:
Yeah, I agree, the other test is enough